Dockerfile

```diff
@@ -8,14 +8,14 @@ ENV NGINX_CLOJURE_VERSION $NGINX_CLOJURE_VERSION
 ENV DEBIAN_FRONTEND noninteractive
 
 # Upgrade the OS and install build dependencies
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y --no-install-recommends \
-            gcc \
-            libpcre3-dev \
-            libssl-dev \
-            make \
-            zlib1g-dev
+RUN apk update && \
+    apk upgrade && \
+    apk add \
```

Comment:
Remove the `update` and `upgrade` commands, and replace `apk add` with `apk add --no-cache`.

See: https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache